opentracing_shim/src/main/java/io/opentelemetry/opentracingshim/ScopeManagerShim.java

```diff
@@ -30,11 +29,9 @@ public ScopeManagerShim(TelemetryInfo telemetryInfo) {
   @Override
   @SuppressWarnings("ReturnMissingNullable")
   public Span activeSpan() {
-    // As OpenTracing simply returns null when no active instance is available,
-    // we need to do an explicit check against DefaultSpan,
-    // which is used in OpenTelemetry for this very case.
+    // OpenTracing simply returns null when the current span is not valid.
     io.opentelemetry.trace.Span span = tracer().getCurrentSpan();
-    if (DefaultSpan.getInvalid().equals(span)) {
+    if (!span.isValid()) {
```

Comment:
Slightly incorrect, as in OT it's possible to set an invalid Span (one with zero-ed ids) as the active instance. We should check against `Span.getInvalid()`

Reply:
Do you think you can give me a unit test that fails with the current code? OTel has many invalid checks, for example in instrumentation - I'd be surprised if we could even support a zero'd but live span in OTel. I'm guessing OT would propagate such a span, but OTel won't - so maybe we can just unify this behavior with the OTel semantics? Anyways a unit test would probably make things very clear, I'm not changing it back if we can't get the current code to fail first :)

Reply:
Another way to phrase my point of confusion - I would think returning a singleton for an invalid span is a performance optimization but wouldn't expect any behavior change. So if the use of a singleton is relied on here, it's unexpected and seems fragile. If it's required I need to add some more detailed docs.

Reply:
Not supporting valid spans with all zero trace *or* span ID is something that irks me about OTel actually https://github.com/open-telemetry/opentelemetry-specification/issues/753